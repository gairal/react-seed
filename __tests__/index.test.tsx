import React from 'react';
import { render } from '@testing-library/react';

import { App } from '@app/index';

const subject = () => render(<App />);

describe('App', () => {
  it('matches snapshot', () => {
    const { asFragment } = subject();

    expect(asFragment()).toMatchSnapshot();
  });
});
