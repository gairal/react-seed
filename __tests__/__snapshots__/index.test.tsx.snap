// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h2>
      React Seed
    </h2>
  </div>
</DocumentFragment>
`;
